<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotControl</class>
 <widget class="qMRMLWidget" name="RobotControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>539</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="collapSafeSwitch">
     <property name="text">
      <string>Safety Switch</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxSafe">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="collapRobotMotion">
     <property name="text">
      <string>Robot Motion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushGetJntAngs">
        <property name="text">
         <string>Get Joint Angles</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushExecute">
        <property name="text">
         <string>Execute Motion To Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushGetEFFPose">
        <property name="text">
         <string>Get End Pose</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushConfirm">
        <property name="text">
         <string>Confirm and Bring Closer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="collapManual">
     <property name="text">
      <string>Manual Adjustment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="1">
       <widget class="QPushButton" name="pushBackward">
        <property name="text">
         <string>Backward</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="ctkSliderWidget" name="sliderRotation"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushFarther">
        <property name="text">
         <string>Farther</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="pushPitch">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelTranslation">
        <property name="text">
         <string>Translation Increment (mm)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushForward">
        <property name="text">
         <string>Forward</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushRoll">
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushCloser">
        <property name="text">
         <string>Closer</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="pushYaw">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushLeft">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushRight">
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="ctkSliderWidget" name="sliderTranslation"/>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="labelRotation">
        <property name="text">
         <string>Rotation Increment (degree)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
